docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -3355,6 +3355,65 @@ RADIUS configuration
   aaa port-access authenticator C4 client-limit 3
   aaa port-access authenticator active
 
+Downloadable ACLs
++++++++++++++++++
+
+HP and Aruba switches running the ArubaOS-Switch operating system (previously called ProVision) support dynamic RADIUS-assigned ACLs. It requires RADIUS authentication using the 802.1X, Web authentication or MAC authentication available on the switch. You can define ACLs in PacketFence so that they can be automatically applied on the ports of the switches based on the role assigned. We have tested it successfully on the Aruba 2930M and 3810 series on version 16.05.0004.
+
+To use this feature, first configure RADIUS and the authentication method on your switch. Next, in the PacketFence web admin interface, go to the 'Configuration' page, and then 'Switches'. Click on the switch you want, then on the 'Roles' tab, and check 'Role by access list'. Now you are able to add ACLs for each role.
```

Comment:
'Configuration' page, and then 'Switches'. => 'Configuration -> Policies and Access Control -> Switches'.